tools/openshift-ci/verification_tests_ci_entrypoint.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/sh
+
+[[ -z "$BUSHSLICER_TEST_ENVIRONMENT" ]] && { echo "BUSHSLICER_TEST_ENVIRONMENT not set"; exit 1; }
+[[ -z "$BUSHSLICER_TEST_CLUSTER" ]] && { echo "BUSHSLICER_TEST_CLUSTER not set"; exit 1; }
```

Comment:
sry, I meant this env var. `BUSHSLICER_TEST_ENVIRONMENT`. Can we get this from `KUBECONFIG`?

Reply:
btw for running tests we need cli tools. They will be missing in this image. Maybe we need to base on `origin` after all, not plain `centos`. Otherwise we need to install `oc` by ourselves.

Reply:
Will investigate getting this value from KUBECONFIG

Reply:
Where origin-cli is based. Do we need to add the centos repositories, etc?

Reply:
Here is its [Dockerfile](https://github.com/openshift/origin/blob/master/images/cli/Dockerfile). It goes all the way back to [`origin-source`](https://github.com/openshift/origin/blob/master/images/source/Dockerfile) which is centos-based.

Reply:
Appropriate thing to do is to build your test image in the `ci-operator` config for your repo, and add in the `oc` from origin-cli using image source

Reply:
@cjryan , seems like `oc` binary was handled by #49. 

The other thing is generating HOSTS spec. Do we need to do anything more for generating hosts here?